src/storage/migration.rs

```diff
@@ -75,9 +75,9 @@ pub fn migrate_data_dir(data_dir: &Path, db_opts: &DbOptions) -> Result<()> {
             migrate_0_24_to_0_26,
         ),
         (
-            VersionReq::parse(">=0.26.0-alpha.1,<0.27.0").expect("valid version requirement"),
-            Version::parse("0.27.0").expect("valid version"),
-            migrate_0_26_to_0_27,
+            VersionReq::parse(">=0.26.0-alpha.1,<0.26.0").expect("valid version requirement"),
+            Version::parse("0.26.0").expect("valid version"),
+            migrate_0_26_alpha_to_0_26,
```

Comment:
Instead of distinguishing between alpha.1 and alpha.2, it would be better to merge these lines into the block above and place `migrate_0_26_alpha_to_0_26_http` inside the `migrate_0_24_to_0_26` function. In that case, please also rename the function to `migrate_0_24_to_0_26_http`.